activitytype-cql/src/main/java/io/nosqlbench/activitytype/cql/statements/binders/RawValueBinder.java

```diff
@@ -0,0 +1,48 @@
+package io.nosqlbench.activitytype.cql.statements.binders;
+
+import com.datastax.driver.core.ConsistencyLevel;
+import com.datastax.driver.core.SimpleStatement;
+import com.datastax.driver.core.Statement;
+import io.nosqlbench.virtdata.core.bindings.ValuesArrayBinder;
+
+import java.util.Optional;
+
+/**
+ * This binder is only meant to be used to directly generate final SimpleStatement query
+ * without use of preparedStatement. This should be avoided any time possible since it's far more
+ * less optimized than use of PreparedStatement. Only Use it when PreparedStatement is not possible
+ */
+public class RawValueBinder
```

Comment:
We should probably use a StringBindingsTemplate to do this for us as a variant of the ReadyCQLStatement.